### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by rcqx**
## Updates:
* A method was utilized to deleted selected items from list (clearList());
* While using clearList() method, index numbers get updated to actual position values 
* Update items object's value for completed key upon user actions.

image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const input = document.getElementById('input');
 let stringData = JSON.stringify(todos.List);
 let listedItem;
 input.addEventListener('keydown', (evt) => {
```
Comment (Whoistolu) [inline src/index.js:12]: - [ ] [REQUIRED] You have done a great implementation, however, there is still something remaining you need to implement. You have ensured that the user is able to mark task completion by selecting the corresponding checkbox, however, you also need to ensure that the updated tasks list will be stored in local storage. It should be updated (`completed`: `true` / `false`) in the local storage. Currently, when a task is marked, the `Completed`: stays as `false` always, it does not get updated.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @jr-cast 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (rcqx) [reply src/index.js:12]: Hiya Whoistolu, I understand. Actually, it is changing true/false but yeah It's not being reflected in localStorage, I'll see to that

Comment (Whoistolu) [reply src/index.js:12]: Alright, kindly fix it to get updated in the local storage.

Comment (peterrobert) [reply src/index.js:12]: This change has been implemented well

**Review by peterrobert — APPROVED** [review]
Hi @jr-cast, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**